src/components/AdminPortal/EventRegistrantsModal/Modal/AddOnSpot/AddOnSpotSuccessModal.tsx

```
@@ -0,0 +1,130 @@
+ * @example
+ * ```tsx
+ * <AddOnSpotSuccessModal
+ *   show={true}
+ *   password="Test@123"
+ *   email="user@example.com"
+ *   attendeeName="John Doe"
+ *   handleClose={() => setShow(false)}
+ * />
+ * ```
+ *
+ * @dependencies
```

ESLint

[error] tsdoc-undefined-tag: The TSDoc tag "@dependencies" is not defined in this configuration

(tsdoc/syntax)